src/main/java/frc/robot/RobotContainer.java

```diff
@@ -204,6 +207,9 @@ private void configureButtonBindings() {
     } /*else {
         buttons[Constants.RTriggerButton].whileHeld(intakeForwardCommand);
       }*/
+    if (intakeDeployCommand != null) {
+      buttons2[Constants.RBumper].whenPressed(intakeDeployCommand);
```

Comment:
Rbumper is already used to run outtake.  Need a better place for this.

Reply:
Deleted completely in favor of an automatic intake deploy and retract command.